.gitignore

```diff
@@ -114,3 +114,7 @@ env/
 
 ### Configuration files ###
 .env
+
+
+### Keys file
+keys.py
```

Comment:
You'll need to document that this file needs to be added manually in the README file